bridgev2/networkinterface.go

```diff
@@ -593,6 +593,52 @@ type GroupCreatingNetworkAPI interface {
 	CreateGroup(ctx context.Context, name string, users ...networkid.UserID) (*CreateChatResponse, error)
 }
 
+type MembershipChangeType string
+
+const (
+	AcceptInvite  MembershipChangeType = "invite,join,1"
```

Comment:
This should probably just be a real struct instead of a struct in a string. It can still have variables for each type (can't do constants though). Non-pointer structs can be compared using `==` too

Reply:
that's what I was initially going to do, but isn't mutability of those structs a problem? Those should really not change...

Reply:
Non-pointer structs are always copied, so there's no way to accidentally mutate the original variable